crates/fuel-core/src/state/rocks_db.rs

```diff
@@ -1218,4 +1206,33 @@ mod tests {
         let _ = open_with_part_of_columns
             .expect("Should open the database with shorter number of columns");
     }
+
+    #[test]
+    fn iter_store__reverse_iterator() {
+        // Given
+        let (mut db, _tmp) = create_db();
+        let value = Arc::new(vec![1, 2, 3]);
```

Comment:
Nit: As we don't use `value` I'd suggest leaving it empty.

```suggestion
        let value = Arc::new(Vec::new());
```

Reply:
Done